BlackJackConsoleApp/Program.cs

```diff
@@ -12,7 +12,10 @@ public static void Main()
             while (blackJack.IsComplete == false)
             {
                 PrintPlayerInfo(blackJack.Computer);
+                Console.WriteLine($"Score: {blackJack.Computer.BlackJackScore}");
                 PrintPlayerInfo(blackJack.Player);
+                blackJack.ChooseAce(blackJack.Player.Cards);
+                Console.WriteLine($"Score: {blackJack.Player.BlackJackScore}");
```

Comment:
Would be nice to make `Console.WriteLine($"Score: {blackJack.Player.BlackJackScore}");` into a method.